dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -874,49 +876,20 @@ STDAPI MrmGetFilePathFromName(_In_opt_ PCWSTR filename, _Outptr_ PWSTR* filePath
 {
     *filePath = nullptr;
 
-    wchar_t path[MAX_PATH];
-    DWORD size = ARRAYSIZE(path);
-    DWORD length = GetModuleFileName(nullptr, path, size);
-    DWORD lastError = GetLastError();
-    std::unique_ptr<wchar_t[]> pathAllocated;
+    wil::unique_cotaskmem_string path;
+    RETURN_IF_FAILED((wil::GetModuleFileNameW<wil::unique_cotaskmem_string, 256>(nullptr, path)));
```

Comment:
Isn't it better to use MAX_PATH instead of 256 here? MAX_PATH may be greater than 256 after a user modification, no?

Reply:
256 is just twice of the default (128). Some people may just don't like MAX_PATH. I doubt anyone is going to modify MAX_PATH. If it does, that's just one more reason to not use MAX_PATH.

Reply:
You're trading codebase readability for a perceived micro-optimization. Other folks coming across this code are going to scratch their heads as to why the initial buffer size was not used here. Something to consider.

Reply:
It's better to use _no_ number here. WIL always produces a heap-allocated string from this method, and the buffer parameter is just a seed to know where to start growing the buffer.  Just don't specify, let WIL do its thing, then there's no Mysterious Constant hanging out here.  (Mysterious Constants in library implementations are fine.)